src/client/client_options.h

```diff
@@ -62,6 +62,10 @@ namespace ClientOptions {
     return bridge_util::Config::getOption<bool>("client.enableDpiAwareness", true);
   }
 
+  inline bool getEnableBridgeApi() {
+    return bridge_util::Config::getOption<bool>("client.enableBridgeApi", true);
```

Comment:
We would appreciate it if you ensure that this is off by default, until we consider it robust enough to turn on for everyone all the time.